modular_skyrat/master_files/code/_globalvars/maint_loot_uncommon.dm

```diff
@@ -10,6 +10,7 @@ GLOBAL_LIST_INIT(uncommon_loot, list(//uncommon: useful items
 		/obj/item/skillchip/glassblowing_master = 10,
 		/obj/item/skillchip/ceramic_master = 10,
 		/obj/item/skillchip/fishing_master = 10,
+		/obj/item/skillchip/brainwashing = 10,
```

Comment:
Very much **NO**, brainwashing has 0, a fat 0 business being obtainable in round without an antag or even research, even if it's 'rare'.